apps/pie-storybook/stories/pie-checkbox.stories.ts

```diff
@@ -112,10 +127,11 @@ const Template = ({
 
     return html`
         <pie-checkbox
-            .value="${ifDefined(value)}"
```

Comment:
I removed this because there was an IDE warning that `ifDefined` has no effect here. It looks like it should only be used for attribute bindings, and this is a property binding:

https://github.com/runem/lit-analyzer/blob/master/docs/readme/rules.md#-no-invalid-directive-binding

Reply:
I just double checked and even with `ifDefined` it submits an empty string if you remove the `"on"` from the storybook control. Looks like we'll have to find another way.